### DISCUSSION TITLE
is it possible to set different AILevel for P1 and P2?

### DISCUSSION THREAD

**Discussion by beterhans** [ikemen-engine/Ikemen-GO#1002] [Q&A] [answered]
for example 

press some key
P1 set to 6
and P2 set to 4?

**Comment by lokpro** [comment, marked as answer]
AFAIK, only through command-line-arguments https://github.com/ikemen-engine/Ikemen-GO/wiki/Miscellaneous-info#command-line-arguments -p<n>.ai <level>: Sets player n's AI to <level>, eg. -p1.ai 8
and only for Quick-VS mode for quick test of your chars.

for example:
Ikemen_GO.exe -p1 kfm -p2 kfm -p3 kfmz -p4 kfmz -p5 kfm -p6 kfm -p7 kfmz -p8 kfmz -p1.ai 1 -p2.ai 1 -p3.ai 1 -p4.ai 1 -p5.ai 8 -p6.ai 8 -p7.ai 8 -p8.ai 8
